src/value/owned.rs

```diff
@@ -547,27 +553,27 @@ mod value_tests {
             }
         ),
         case::list(
-            OwnedValue::List(OwnedSequence::new(vec![OwnedElement::from(OwnedValue::Boolean(true)), OwnedElement::from(OwnedValue::Boolean(false))])),
+            OwnedValue::List(vec![OwnedValue::Boolean(true).into(), OwnedValue::Boolean(false).into()].into_iter().collect()),
```

Comment:
Not sure if it is worth it with only two elements, but `into_iter().map(|v| v.into()).collect()` might be be a bit more concise.